src/tree_schema.h

```diff
@@ -139,7 +139,7 @@ struct ly_set;
  * - ::lysp_feature_next()
  */
 
-/**
+/*
```

Comment:
Probably a typo.

Reply:
not a typo, but forgotten debugging change, fixed